.github/workflows/release-hosts.yaml

```diff
@@ -0,0 +1,24 @@
+name: Release MacOS hosts
+
+on:
+  workflow_dispatch: {}
+  schedule:
+    - cron: 0 10 * * *
+
+jobs:
+  release_hosts:
+    secrets:
+      AWS_ACCOUNT_ID: ${{ secrets.AWS_ACCOUNT_ID }}
+    steps:
+      - uses: actions/checkout@v3
+      # TODO use aws ssm
```

Comment:
is this to be done in this PR or the next one? why is aws ssm better?

Reply:
In next step